index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200&family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="sub-container">
+      <nav class="navbar">
+        <a href="#" class="menu-icon">
+          <img src="./images/my-logo/logo3.jpg" alt="logo" width="35"
+        /></a>
+        <div class="menu">
+          <img src="./images/menu/Icon-Menu.png" alt="menu icon" />
+        </div>
+      </nav>
+    </header>
+    <section class="main-container">
+      <div class="logo-intro">
+        <img
+          src="./images/my-logo/logo-1.png"
+          alt="logo"
+          width="170"
+          class="my-image"
+        />
+        <h1 class="greeting">
+          Nice to meet you, <br />
+          I'm John
+        </h1>
+      </div>
+      <div class="sub-main-container">
+        <div class="intro">
+          <h3 class="profession">A Web Developer</h3>
+          <p class="welcome">
+            I can help you build a product , feature or <br />
+            website Look through some of my work <br />
+            and experience! If you like what you see <br />
+            and have a project you need coded,
+          </p>
```

Comment:
- [ ] Please kindly implement the changes requested by the previous reviewer by wrapping your intro to `section` tag to follow HTML best practices and consistency in your code .